openbb_platform/dev_install.py

```diff
@@ -192,7 +193,8 @@ def install_platform_cli():
         CMD = [sys.executable, "-m", "poetry"]
 
         subprocess.run(
-            CMD + ["lock", "--no-update"],
+            CMD + ["lock"],
+            # CMD + ["lock", "--no-update"],
```

Comment:
leftover?

Reply:
I might be able to restore that now that I've cleared and rebuilt the Github Actions cache..

Reply:
Nope, experiment over.